### PR TITLE
site css completed ready for code review

### PR DESCRIPTION
**Pull request by HazDuck**
-added css to site page 
-saved assets

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -1,3 +1,92 @@
...
+    text-shadow: 0 0 2.5px #fff, 0 0 5px #fff, 0 0 7.5px #fff, 0 0 20px #FF1177, 0 0 35px #FF1177, 0 0 40px #FF1177, 0 0 50px #FF1177, 0 0 75px #FF1177;
+    color: #FFFFFF;
+    font-family: "Monoton", sans-serif;
+    letter-spacing: 3px;
+    text-transform: uppercase;
+    margin: 40px 30px 20px 0;
+}
+
+aside {
+    height: 600px;
```
Comment (katestarks) [inline css/styles.css:32]: Doesn't do anything?

File: css/styles.css [hunk 2]
```diff
@@ -1,3 +1,92 @@
...
+}
+
+aside div {
+    width: 70px;
+    height: 70px;
+    border: 8px solid white;
+    border-radius: 40px;
+    margin: 0 auto 15px auto;
+    background-size: cover;
+    background-position: center;
```
Comment (katestarks) [inline css/styles.css:55]: Doesn't do anything

File: css/styles.css (hunk 1 above)
Comment (HazDuck) [reply css/styles.css:32]: done

File: css/styles.css (hunk 2 above)
Comment (HazDuck) [reply css/styles.css:55]: done

**Pull request merged**